src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,20 @@
+package duke.exception;
+
+/**
+ * Represents an exception class that can be thrown when using Duke.
+ *
+ * @author Terng Yan Long
+ */
+public class DukeException extends RuntimeException {
+    /**
+     * Initialises a new instance of DukeException.

```

Comment:
I think the term for creating a new instance of a class is instantiate instead of initialize.